controllers/account.js

```diff
@@ -40,29 +40,36 @@ module.exports.renderChangeProfile = async (req, res) => {
 };
 
 module.exports.updatePassword = async (req, res) => {
-  if (
-    !req.body.oldPassword ||
-    !req.body.newPassword ||
-    !req.body.repeatPassword
-  ) {
+  const { oldPassword, newPassword, repeatPassword } = req.body;
+
+  console.log(oldPassword);
+  console.log(newPassword);
+  console.log(repeatPassword);
+  if (!oldPassword || !newPassword || !repeatPassword) {
     res.redirect("/account/change-password");
   }
   User.findById(req.user._id, (err, user) => {
     if (err) {
       res.send(err);
     } else {
-      if (req.body.newPassword === req.body.repeatPassword) {
-        user.changePassword(
-          req.body.oldPassword,
-          req.body.newPassword,
-          (err) => {
-            if (err) {
-              res.redirect("/account/change-password?feedback=incorrectPw");
-            } else {
-              res.redirect("/account?feedback=pwChangeSuccess");
-            }
+      if (newPassword === repeatPassword) {
+        user.changePassword(oldPassword, newPassword, (err) => {
+          if (err) {
+            res.send(
+              JSON.stringify({
+                message: "Incorrect old password",
+                style: "message-box-alert",
```

Comment:
I don't think that's a good way tbh, backend shouldn't be sending class names for frontend. I would recommend instead to send a message code instead like:

```
{
  "code": "incorrect_old_password"
}
```

Then read the message code in frontend and map the it accordingly to what code you receive.